src/browser/pages/vscode.ts

```diff
@@ -19,7 +17,18 @@ type NlsConfiguration = {
   _resolvedLanguagePackCoreLocation?: string
   _corruptedFile?: string
   _languagePackSupport?: boolean
-  loadBundle?: any
+  loadBundle?: FixMeLater
+}
+
+/**
+ * Helper function to create the path to the bundle
+ * for getNlsConfiguration.
+ */
+export function createBundlePath(_resolvedLanguagePackCoreLocation: string, bundle: string) {
+  // NOTE@jsjoeio - this comment was here before me but not sure
+  // why it's here or how to fix.
+  // FIXME: Only works if path separators are /.
```

Comment:
Refers to operating systems that use a different path separator. Probably just Windows although I'm not 100% sure using `/` here breaks Windows or not.

Reply:
Ahhh! Got it. Okay, I can update the comment then with those details. Any suggestions on how I could write a test for this function to see how it behaves on Windows?

Reply:
I'm not really sure how effective unit tests will be here tbh, I think we'll need to do in-browser tests on different platforms.

Reply:
Can we get away with `path.join` as an OS agnostic path builder?

Reply:
^that would be awesome! I will leave for now and eventually we can follow-up later!